lib/atat-web-api-stack.ts

```diff
@@ -62,11 +72,67 @@ export class AtatWebApiStack extends cdk.Stack {
       apiProps.vpcConfig = {
         vpc: network.vpc,
         interfaceEndpoint: network.endpoints.apigateway,
-        tempDontUseVpcEndpoint: true,
       };
     }
 
     const api = new AtatRestApi(this, "HothApi", apiProps);
+    if (props.apiDomain && network) {
+      const certificate = acm.Certificate.fromCertificateArn(this, "ApiCertificate", props.apiDomain.acmCertificateArn);
+      /* eslint-disable spaced-comment */
+      // Toggle between the ALB or the NLB by adding (ALB) or removing (NLB) the leading `/` on
+      // the following line. This is a _temporary_ functionality; only one will remain in the
+      // code base after this PR is merged.
+      //*
```

Comment:
This is a really neat toggle by removing the `/`. I had to actually do it to fully understand 😆